### ISSUE TITLE
Fix clients.xhtml skipLines in irc example

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6433]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#6433|
|Type|defect|
|Created|2013-04-14 06:23:13Z|
|Branch|https://github.com/twisted/twisted/tree/client-skiplines-6433|

document-words-examples-5589 changed the __doc__ length of a few modules, including ircLogBot.py, which, combined with the fact that the client howto uses skipLines, caused the example to be misrendered.

Attachments:

* [fix-client-skipLines.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3ee/3eee834d9a1d5a444e7175a0c9dbc0260ec81986/1ff28fe29e96555d86440bbcc9e77de1d21c75d1.patch) (673 bytes) - added by Julian on 2013-04-14 06:23:32Z - 
* [fix-client-skipLines.2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/3ee/3eee834d9a1d5a444e7175a0c9dbc0260ec81986/f0711fb1aa8435442ab0a4d8c80cf2735ae7ae5c.patch) (683 bytes) - added by Julian on 2013-04-15 00:19:54Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6433 6433
type__defect defect
reporter__Julian Julian
priority__normal normal
milestone__ 
branch__branches_client_skiplines_6433 branches/client-skiplines-6433
branch_author__Julian Julian
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1365920593000000 1365920593000000
changetime__1366297363000000 1366297363000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @Julian|
|-|-|

Thanks.

I think the `skipLines` feature of Lore is pretty awful.  It can't really help but be extremely error prone.  The only reason we aren't constantly fixing bugs like this one is that almost no one ever uses the `skipLines` features.  I suggest removing its usage here as well.

**Comment by twisted-trac** [comment]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)<a name="note_2"></a>|@Julian commented|
|-|-|

I agree, and I considered removing it, the reason I kept it is that this is a fairly basic howto, with not much to do in particular with `t.words`, but the module docstring is 29 lines of comment, which is a bit much to splat into there.

**Comment by twisted-trac** [comment]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)<a name="note_3"></a>|@Julian removed owner|
|-|-|

Here's a removal. Merger can pick whichever.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [38149]) Branching to 'client-skiplines-6433'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

(In [38150]) apply fix-client-skipLines.2.patch, add news file. refs [#6433](https://github.com/twisted/twisted/issues/6433)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch. Used the second one and forced a [build](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/client-skiplines-6433). Will merge when its green.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38155]) Merge client-skiplines-6433: Remove usage of skipLines in IRC example to avoid misrendering of example code.

Author: Julian
Reviewer: exarkun, thijs
Fixes: [#6433](https://github.com/twisted/twisted/issues/6433)
